catalog/resource_registered_model.go

```diff
@@ -0,0 +1,78 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceRegisteredModel() *schema.Resource {
+	s := common.StructToSchema(
+		catalog.CreateRegisteredModelRequest{},
+		func(m map[string]*schema.Schema) map[string]*schema.Schema {
```

Comment:
Oh I see we document that `comment` is supported - does that work if it's not part of the Schema defined here?

Reply:
Yup it works since it's part of `catalog.CreateRegisteredModelRequest`! The schema attributes you see here simply add to the schema defined there. You can see examples of `comment` working in the tests